src/System.Runtime.Serialization.Xml/tests/SerializationTestTypes/ObjRefSample.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace SerializationTestTypes
+{
+    #region Simple Class Structs with Ref
+
+    [DataContract(IsReference = true)]
+    public class SimpleDC
+    {
+        [DataMember]
+        public string Data;
+        public SimpleDC() { }
+        public SimpleDC(bool init)
```

Comment:
It was referenced by other DCS tests which are going to port into core. So I keep it there.